sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/DocumentCrudTest.java

```diff
@@ -315,17 +335,121 @@ public void upsertDocument_ReplaceDocument(String documentId) throws Throwable {
 
         // Replace document
 
-        Mono<CosmosAsyncItemResponse> readObservable = container.upsertItem(properties, new CosmosItemRequestOptions());
+        Mono<CosmosAsyncItemResponse<CosmosItemProperties>> readObservable = container.upsertItem(properties, new CosmosItemRequestOptions());
         System.out.println(properties);
 
         // Validate result
 
-        CosmosResponseValidator<CosmosAsyncItemResponse> validator = new CosmosResponseValidator.Builder<CosmosAsyncItemResponse>()
+        CosmosResponseValidator<CosmosAsyncItemResponse<CosmosItemProperties>> validator = new CosmosResponseValidator.Builder<CosmosAsyncItemResponse<CosmosItemProperties>>()
             .withProperty("newProp", newPropValue).build();
 
         validateSuccess(readObservable, validator);
     }
 
+    @Test(groups = {"simple"}, timeOut = TIMEOUT)
+    public void typedItemsTest() throws Throwable {
```

Comment:
to "Test" suffix or prefix in the test name to look the same as other test methods

Reply:
Sure, always overlook this :-). Removed